client/views/root/canada_post.js

```diff
@@ -0,0 +1,3 @@
+import RootView from '../generic-settings-form';
```

Comment:
We'll see. If I can make the Shipping Label dialog fully data-driven, I'll change this name to something less specific. If not, we'll need an entry point just for the labels.